### PR TITLE
fixed #51 (use of properly-level examples)

### LINKED ISSUE #51: Add support for individual examples
In essence, address the limitation currently detailed in README:
> This module only uses the example (or examples) property inside a given resource/method body. It does not take into consideration any of the example properties defined inside the properties or schema of the body itself.

### PR DESCRIPTION
**Pull request by brevity**
Closes #51

### CODE REVIEW

File: test/test08.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ describe('osprey mock service v0.8', function () {
       return popsicle.default('/api/test')
         .use(server(http))
         .then(function (res) {
-          expect(JSON.parse(res.body)).to.deep.equal({success: true})
+          // expect(JSON.parse(res.body)).to.deep.equal({success: true})
```
Comment (jstoiko) [inline test/test08.js]: ?

File: osprey-mock-service.js [hunk 2]
```diff
@@ -133,17 +141,19 @@ function handler (method) {
...
       if (Array.isArray(body.examples)) {
         example = []
-
         body.examples.forEach(function (ex) {
-          var obj = {}
-          obj[ex.name] = ex.structuredValue
-          example.push(obj)
+          example.push(ex.structuredValue)
         })
+        example = example[Math.floor(Math.random() * example.length)]
```
Comment (jstoiko) [inline osprey-mock-service.js]: can't we re-use `getSingleExample()` here? it is used only for headers examples for some reason but the (example(s)) logic is the same for both headers and bodies so I think we should re-use it.

**Review by ForsakenHarmony — APPROVED** [review]
LGTM

**Review by jstoiko — APPROVED** [review]
made a few tweaks

**Pull request merged**